packages/webview_flutter/webview_flutter/example/integration_test/webview_flutter_test.dart

```diff
@@ -58,9 +58,9 @@ void main() {
       ),
     );
     final WebViewController controller = await controllerCompleter.future;
-    await controller.loadUrl('https://www.google.com/');
+    await controller.loadUrl('https://www.example.com/');
```

Comment:
Should this URL be a `const String` at the top of the test files, so it can be changed more easily in the future?

Reply:
Since there's no actual relation between its use in the different tests (i.e., we could make it different in every test with no issue) I would lean away from extracting a shared constant. Maybe within each test, but that doesn't change things much.